app/src/main/java/com/jnj/vaccinetracker/register/dialogs/HistoricalVisitDateDialog.kt

```diff
@@ -72,8 +72,16 @@ class HistoricalVisitDateDialog() : BaseDialogFragment() {
    }
 
    private fun setupDatePicker() {
-      val c = Calendar.getInstance()
-      datePicker.maxDate = c.timeInMillis
+      val calendar = Calendar.getInstance()
+      calendar.add(Calendar.DAY_OF_YEAR, -1)
+      datePicker.maxDate = calendar.timeInMillis
+      //  historical visits are restricted to dates strictly before today.
+      datePicker.updateDate(
```

Comment:
Is it required? we just want set "yesterday" as max date in calendar - looks like this "calendar.add(Calendar.DAY_OF_YEAR, -1)" does the trick and rest of code is not needed? but correct me if I'm wrong

Reply:
Yes this is required in the sense that we set the default visible date to the max date (yesterday), ensuring that the date picker doesn't default to an invalid date.